src/common/interval_map.h

```diff
@@ -245,6 +245,11 @@ class interval_map {
     const_iterator &operator*() {
       return *this;
     }
+    const bool contains(K _off, K _len) const {
```

Comment:
The first 'const' does not make sense.
(but a constexpr would)

Reply:
Thanks, I will address.   This is just a learning-C++ thing ;)